nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -80,6 +82,13 @@ CollisionMonitor::on_configure(const rclcpp_lifecycle::State & state)
 
   collision_points_marker_pub_ = this->create_publisher<visualization_msgs::msg::MarkerArray>(
     "~/collision_points_marker");
+  
+  // Toggle service initialization
+  const std::string service_prefix = get_name() + std::string("/");
```

Comment:
You can just do `~/toggle` like we do `~/collision_points_marker` above. You don't need to do this `get_name()` or prefix.